### ISSUE TITLE
[BUG] bucket event error

### ISSUE THREAD

**Issue by stratoru** [arkayenro/arkinventory#1658]
**What version number you are using?** 
3.10.04-alpha-16

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
WotLK

**What language is the game client set to?**
English

**Describe the bug**
Latest update threw single error as below. Item I looted is not showing in bag, not even when searching for it via ArkInventory.

**To Reproduce**
Steps to reproduce the behaviour, if possible:
Loot item. Check bag. Item is missing.

**Screenshots**

**Additional context**
BugSack:

```
1x ArkInventory\Core\ArkInventory-31004.16.lua:2951: Usage: RegisterBucket(event, interval, callback): 'callback' - method not found on target object.
[string "=[C]"]: ?
[string "@ArkInventory\Externals\Ace3\AceBucket\AceBucket-3.0-4.lua"]:142: in function <...Inventory\Externals\Ace3\AceBucket\AceBucket-3.0.lua:122>
[string "=(tail call)"]: ?
[string "@ArkInventory\Core\ArkInventory-31004.16.lua"]:2951: in function <ArkInventory\Core\ArkInventory.lua:2900>
[string "=[C]"]: ?
[string "@ArkInventory\Externals\Ace3\AceAddon\AceAddon-3.0-13.lua"]:66: in function <...rkInventory\Externals\Ace3\AceAddon\AceAddon-3.0.lua:61>
[string "@ArkInventory\Externals\Ace3\AceAddon\AceAddon-3.0-13.lua"]:523: in function `EnableAddon'
[string "@ArkInventory\Externals\Ace3\AceAddon\AceAddon-3.0-13.lua"]:626: in function <...rkInventory\Externals\Ace3\AceAddon\AceAddon-3.0.lua:611>
```

**Comment by arkayenro** [comment]
should be fixed next release

**Comment by arkayenro** [comment]
fixed in 3.10.05
